src/main/org/nlogo/prim/etc/_extracthsb.scala

```diff
@@ -0,0 +1,99 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.prim.etc
+
+import org.nlogo.api, api.{ Syntax, LogoListBuilder, LogoList }
+import org.nlogo.nvm.{ Context, Pure, Reporter }
+
+object extracthsb {
```

Comment:
I didn't have a good grasp of what belongs in Color, and what belongs here.  I just kind of followed whatever _extracthsb.java did.  Is there a good decision procedure for this stuff?